bodhi/tests/server/functional/test_generic.py

```diff
@@ -111,18 +111,49 @@ def test_markdown(self):
         res = self.app.get('/markdown', {'text': 'wat'}, status=200)
         self.assertEquals(
             res.json_body['html'],
-            "<div class='markdown'>"
-            "<p>wat</p>"
-            "</div>"
+            '<div class="markdown">'
+            '<p>wat</p>'
+            '</div>'
```

Comment:
The changing of the way we clean the output meant that the quotes around the class="markdown" changed in the output generated by bleach. which made the tests fail. Do you want me to just change those lines, and do the other quotes style changes on the other lines in a seperate commit